tests/VObject/TimeZoneUtilTest.php

```diff
@@ -8,8 +8,8 @@ class TimeZoneUtilTest extends TestCase
 {
     public function setUp(): void
     {
-        // clearing the tz cache
-        TimeZoneUtil::$map = null;
+        // clearning the tz cache
```

Comment:
```suggestion
        // cleaning the tz cache
```

Do we even need this comment? It says no more than what the code is "obviously" (to me) doing.